<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNew500"/>
     <addaction name="actionNew1000"/>
     <addaction name="actionNew4000"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuBrush_Texture">
     <property name="title">
      <string>Brush Texture</string>
     </property>
     <addaction name="actionBrush_Texture_1"/>
     <addaction name="actionBrush_Texture_2"/>
     <addaction name="actionBrushCustom"/>
    </widget>
    <addaction name="actionChange_a_Pen_Size"/>
    <addaction name="actionPen_Color"/>
    <addaction name="actionBrush_Transparency"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Image"/>
    <addaction name="actionClear_Color"/>
    <addaction name="separator"/>
    <addaction name="actionBrush_Circle"/>
    <addaction name="menuBrush_Texture"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionChange_a_Pen_Size">
   <property name="text">
    <string>Brush Size</string>
   </property>
  </action>
  <action name="actionClear_Image">
   <property name="text">
    <string>Clear Image</string>
   </property>
  </action>
  <action name="actionPen_Color">
   <property name="text">
    <string>Brush Color</string>
   </property>
  </action>
  <action name="actionBrush_Circle">
   <property name="text">
    <string>Brush Circle</string>
   </property>
  </action>
  <action name="actionBrush_Transparency">
   <property name="text">
    <string>Brush Transparency</string>
   </property>
  </action>
  <action name="actionClear_Color">
   <property name="text">
    <string>Clear Color</string>
   </property>
  </action>
  <action name="actionNew500">
   <property name="text">
    <string>New 500 x 500</string>
   </property>
  </action>
  <action name="actionNew1000">
   <property name="text">
    <string>New 1000 x 1000</string>
   </property>
  </action>
  <action name="actionNew4000">
   <property name="text">
    <string>New 4000 x 4000</string>
   </property>
  </action>
  <action name="actionBrush_Texture_1">
   <property name="text">
    <string>Brush Texture 1</string>
   </property>
  </action>
  <action name="actionBrush_Texture_2">
   <property name="text">
    <string>Brush Texture 2</string>
   </property>
  </action>
  <action name="actionBrushCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
